js/src/components/SEMrushRelatedKeyphrasesModal.js

```diff
@@ -83,10 +115,19 @@ class SEMrushRelatedKeyphrasesModal extends Component {
 							className="yoast-gutenberg-modal__content yoast-related-keyphrases-modal__content"
 						>
 							<Slot name="YoastRelatedKeyphrases" />
-
 						</ModalContainer>
 					</Modal>
 				}
+				{ ! isLoggedIn && <div className={ "yoast" }>
+					<ButtonStyledLink
+						variant={ "secondary" }
+						id={ `yoast-get-related-keyphrases-${location}` }
+						href={ "https://oauth.semrush.com/auth/login" }
+						onClick={ this.onLinkClick }
+					>
+						{ __( "Get related keyphrases", "wordpress-seo" ) }
```

Comment:
We should add a visually hidden text to inform users this link will open a new browser _window_ (actually: the popup).

Reply:
Done.